### ISSUE TITLE
Ghost activity info reversed?

### ISSUE THREAD

**Issue by DeceptivePastry** [azsry/phasmophobia_mechanics#9]
I think this bit of info may be backward:

"If a random number from 0 to <random ability value> (based on difficulty) is greater than or equal to their current activity level, including all ghost-specific multipliers, and an additional 15% if playing solo, they have a chance of interacting with objects, or going to their favourite room, appear, use the fusebox, etc.

This means that a higher Ghost activity level, Oni/Wraith multiplier, and playing solo decreases the chance of the ghost interacting with the house"

In IdleState it shows:
```
float num2 = Mathf.Clamp((float)num + this.ghostAI.ghostInfo.activityMultiplier + (float)this.WraithMultiplier + (float)this.OniMultiplier + (float)((PhotonNetwork.playerList.Length == 1) ? 15 : 0), 0f, 100f);

if ((float)Random.Range(0, this.maxRandomAbilityValue) <= num2 && Random.Range(0, 2) == 1)
```

So it looks like it's actually if random number is **less than** or equal to their current activity level, no? Which would mean a higher activity multiplier increases chance of activity? This is further supported by the fact that Onis get an activity multiplier of 30 when > 0 people are in their room, and in-game they very clearly get much more active when someone is in their room.

**Comment by azsry** [comment]
Yep, you're correct. I must have written that part while I was sleepy. I'll update the guide accordingly to fix these issues. Thanks for spotting this!

**Comment by Mythra** [comment]
Hey! I'm going through the repo, and closing out old stale issues with the new update coming out. Seems this was solved, and updated correctly, so I'm going to go ahead, and close this out. Thanks!
